### ISSUE TITLE
Moving/deleting symboly ids is impossible

### ISSUE THREAD

**Issue by dawehner** [unisonweb/unison#763]
## Steps to reproduce

* Define a new function:

```
-- Any operator
(++) xs ys = xs
```

* Now you realize you actually want to give it a nice name: 
```
.> move.term ++ append

  InvalidWordyId ""

.> move.term (++) append

  InvalidWordyId ""
```
but there seems to be no way to achieve this yet.

The same problem appears on aliases and deletion as well.

**Comment by aryairani** [comment]
As a workaround, you should be able to move the term to some namespace other than the root by specifying some kind of ; e.g.:
```
.pkg1> move.term ++ .pkg1.append`
```
or
```
.> move.term pkg1.++ pkg2.++
```

/cc @stacycurl

**Comment by dawehner** [comment]
Thank you for the explanation

**Comment by seagreen** [comment]
I just hit this one too. I see that [wordyId0](https://github.com/unisonweb/unison/blob/40afe19f5520769d3b2b35d5cd1aa06baab9533c/parser-typechecker/src/Unison/Lexer.hs#L624) is the place that's failing with `InvalidWordyId`.

I'm guessing here, but any chance the problem is that [Unison.Codebase.Path](https://github.com/unisonweb/unison/blob/40afe19f5520769d3b2b35d5cd1aa06baab9533c/parser-typechecker/src/Unison/Lexer.hs#L445) needs to be made more liberal so that it parses both wordyIds and symbols?

**Comment by seagreen** [comment]
Also will you explain the trick again? I'm not having any luck:
```
.> ls

  1. +++      (Text -> Text -> Text)
  2. base/    (470 definitions)
  3. builtin/ (278 definitions)
  4. ian/     (96 definitions)
  5. runar/   (80 definitions)

.> move.term 1 append

  InvalidWordyId ""

.> move.term .+++ .append

  InvalidWordyId ""

.> move.term +++ append

  InvalidWordyId ""
```

**Comment by mitchellwrosen** [comment]
@seagreen Looks like you can't do @aryairani's trick in the root namespace. How about this?
```
fork . temp
move.term .temp.+++ .temp.append
merge .temp
```

**Comment by runarorama** [comment]
Potentially fixed by #1432

**Comment by aryairani** [comment]
> Potentially fixed by #1432

I tried out the specific scenario in the ticket description, and it was indeed fixed by #1432!
